ONBOARDING.md

```
@@ -1,11 +1,11 @@
 # Onboarding Guide
 
-Client side set up is straight forward and standard, i.e clone the repo [Resonte Client Side Repo](https://github.com/AOSSIE-Org/Resonate), do `flutter pub get` in the root of the project etc so this guide would focus on the backend env set up.
+Client-side set-up is straightforward and standard, i.e., clone the repo [Resonate Client Side Repo](https://github.com/AOSSIE-Org/Resonate), do `flutter pub get` in the root of the project, etc. This guide would focus on the backend env setup.
 
-Please strictly stick to the guide do not go off installing stuff on your own
+Please strictly stick to the guide; do not go off installing stuff on your own.
 
 > #### **Setting up Firebase for FCM**
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)